### ISSUE TITLE
No module named 'gradio' error

### ISSUE THREAD

**Issue by artthili** [cocktailpeanut/audiogradio.pinokio#1]
when I run  audiogradio after installation I always get error as below,

(env) E:\Pinokio\api\audiogradio.pinokio.git\audiocraft>python demos\musicgen_app.py
Traceback (most recent call last):
  File "E:\Pinokio\api\audiogradio.pinokio.git\audiocraft\demos\musicgen_app.py", line 21, in <module>
    import gradio as gr
ModuleNotFoundError: No module named 'gradio'

then nothing happened and stop on 3/6 steps

**Comment by wptechnology** [comment]
Same Here, works properly on Mac M1 but On Windows, GRadio error at step 3/6 then nothing happend...

**Comment by mandaliyavatsal** [comment]
Same hear. I'm using M1 Macbook Pro with macos sonoma

**Comment by dseeker** [comment]
same here, Apple M3 Max running Sonoma 14.5
[logs.zip](https://github.com/user-attachments/files/15856225/logs.zip)

**Comment by machoretro** [comment]
Yup this is still a thing, windows 11, manually installed in in the env got me to the next part missing.
pinokio\api\audiogradio.pinokio.git\audiocraft\demos\audiogen_app.py", line 23, in <module>
    from audiocraft.data.audio_utils import convert_audio
ModuleNotFoundError: No module named 'audiocraft'

**Comment by pinger00** [comment]
+1 Windows 10

**Comment by chiangs** [comment]
Mac M1, new install. getting the no modue named "gradio". any tips to get this working?

**Comment by vladislav-vitalyevich-panin** [comment]
Can confirm for Windows 11, fresh install.

**Comment by vladislav-vitalyevich-panin** [comment]
Ok, so, from digging around in the environment, it seems that the produced environment is completely broken.

First, there are no gradio and audiocraft modules.

This would be fixable by manually following the steps from the install json file, however the CUDA version that is being installed is incompatible with the torch version - both come from the same definition files in this repository.

I wonder if this happened because the versions in requirements.txt are not properly set (not pinned). Maybe there is a configuration that will work, but with the astronomical amount of permutations I've given up after 20 minutes.
